src/components/Details/DetailDescription.tsx

```diff
@@ -0,0 +1,220 @@
+        <div key="service-icon" className={iconStyle}>
+          <Tooltip position={TooltipPosition.top} content={<>Application</>}>
+            <Badge className={'virtualitem_badge_definition'}>A</Badge>
+          </Tooltip>
+        </div>
+        <span>
+          <Link to={'/namespaces/' + namespace + '/applications/' + appName}>{appName}</Link>
+        </span>
+      </li>
+    );
+  }
+
+  private renderServiceItem(namespace: string, serviceName: string) {
+    return (
+      <li key={`Service_${serviceName}`}>
+        <div key="service-icon" className={iconStyle}>
+          <Tooltip position={TooltipPosition.top} content={<>Service</>}>
+            <Badge className={'virtualitem_badge_definition'}>S</Badge>
+          </Tooltip>
+        </div>
+        <span>
+          <Link to={'/namespaces/' + namespace + '/services/' + serviceName}>{serviceName}</Link>
+        </span>
+      </li>
+    );
+  }
+
+  private renderEmptyItem(type: string) {
+    const message = 'No ' + type + ' found';
+    return <div> {message} </div>;
+  }
+
+  private appList() {
+    const applicationList =
+      this.props.apps && this.props.apps.length > 0
+        ? this.props.apps.map(name => this.renderAppItem(this.props.namespace, name))
+        : this.renderEmptyItem('applications');
+
+    return [
+      <div key="service-list" className={resourceListStyle}>
+        <Title headingLevel="h5" size="lg" className={titleStyle}>
+          Applications
+        </Title>
+        <ul style={{ listStyleType: 'none' }}>{applicationList}</ul>
+      </div>
+    ];
+  }
+
+  private renderWorkloadItem(workload: AppWorkload) {
+    return (
+      <span>
+        <div key="service-icon" className={iconStyle}>
+          <Tooltip position={TooltipPosition.top} content={<>Workload</>}>
+            <Badge className={'virtualitem_badge_definition'}>W</Badge>
+          </Tooltip>
+        </div>
+        <Link to={'/namespaces/' + this.props.namespace + '/workloads/' + workload.workloadName}>
+          {workload.workloadName}
+        </Link>
+        {!workload.istioSidecar && (
+          <MissingSidecar namespace={this.props.namespace} tooltip={true} style={{ marginLeft: '10px' }} text={''} />
+        )}
+      </span>
+    );
+  }
+
+  private renderWorkloadHealthItem(sub: HealthSubItem) {
+    let workload: AppWorkload | undefined = undefined;
+    if (this.props.workloads && this.props.workloads.length > 0) {
+      for (let i = 0; i < this.props.workloads.length; i++) {
+        if (sub.text.startsWith(this.props.workloads[i].workloadName)) {
+          workload = this.props.workloads[i];
+          break;
+        }
+      }
+    }
+    if (workload) {
+      return (
+        <span>
+          <div key="service-icon" className={iconStyle}>
+            <Tooltip position={TooltipPosition.top} content={<>Workload</>}>
+              <Badge className={'virtualitem_badge_definition'}>W</Badge>
+            </Tooltip>
```

Comment:
Please replace this with the new PFBadge:

`<PFBadge badge={PFBadges.Workload} position={TooltipPosition.top} />`